libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -792,7 +792,7 @@ AP_AHRS_DCM::drift_correction(float deltat)
     // flat, but still allow for yaw correction using the
     // accelerometers at high roll angles as long as we have a GPS
     if (AP_AHRS_DCM::use_compass()) {
-        if (have_gps() && is_equal(gps_gain,1.0f)) {
+        if (have_gps() && is_equal<float>(gps_gain,1.0f)) {
```

Comment:
What I did here: casts and is_equal<float> calls were converted to something like: `is_equal(gps_gain.get(), 1.0f)` - because it's better to call the get() method explicitly than playing with the template.